setup.py

```diff
@@ -34,7 +34,7 @@ def unique_flatten_dict(d):
     'networkx': ['networkx>=2.5'],
     'gremlin': ['gremlinpython'],
     'bolt': ['neo4j', 'neotime'],
-    'nodexl': ['openpyxl', 'xlrd'],
+    'nodexl': ['openpyxl==3.1.0', 'xlrd'],
```

Comment:
do `<=3.1.0` ? in case they already have a working version

Reply:
maybe pip supports `<=3.1.0,>=3.1.1` ? i'm not 100% here

alt: keep unpinned here, and move to the gha?

Reply:
still to do

also: 3.1.2 is out